whois_rdap.json

```diff
@@ -646,5 +634,37 @@
             "output": [],
             "versions": "EQ(*)"
         }
-    ]
-}
+    ],
+    "pip_dependencies": {
```

Comment:
Can we get rid of those?

Reply:
They were added by the dependencies wheel-building tool.